src/main/java/net/pms/dlna/RootFolder.java

```diff
@@ -112,7 +117,7 @@ public void discoverChildren() {
 				PMS.getConfiguration().getDataFile("UMS.last"),
 				PMS.getConfiguration().getInt("last_play_limit", 250),
 				Playlist.PERMANENT|Playlist.AUTOSAVE);
-			addChild(recentlyPlayed);
+			addChild(last, true, isAddGlobally);
```

Comment:
@SubJunk why is not here `recentlyPlayed`

Reply:
I don't remember getting a conflict at that line *shrugs*